graylog2-web-interface/src/routing/Routes.ts

```diff
@@ -185,6 +186,8 @@ const Routes = {
     };
 
     if (rangeType && timeRange) {
+      queryParams.rangetype = rangeType;
```

Comment:
I think this is not necessary. We do deduce the type of the time range passed through query parameters by the presence of the corresponding fields (`relative` vs. `from`/`to` vs. `keyword`).

Reply:
That logic may need some work then, because I tried leaving it out first and I couldn't get `absolute` time ranges to work without the inclusion of `rangetype`. `relative` works fine (I used it in a similar feature for Sigma rules) but `absolute` defaults to the relative 5 minutes ago search if `&rangetype=absolute` is not provided as a parameter.

Reply:
@dennisoelkers would you mind checking this out again and seeing if this is working as you thought?